akka-docs/src/main/paradox/serialization.md

```diff
@@ -36,13 +36,13 @@ Akka itself uses Protocol Buffers to serialize internal messages (for example cl
 ### Configuration
 
 For Akka to know which `Serializer` to use for what, you need to edit your configuration: 
-in the "akka.actor.serializers"-section, you bind names to implementations of the `akka.serialization.Serializer`
+in the `akka.actor.serializers`-section, you bind names to implementations of the @scala[@apidoc[akka.serialization.Serializer](Serializer)]@java[@apidoc[akka.serialization.JSerializer](JSerializer)]
```

Comment:
Actually I think `akka.serialization.Serializer` is correct both for Scala and Java: you'd use `JSerializer` to create a custom serializer in Java, but you don't necessarily bind to custom serializers. Since `JSerializer` extends `Serializer` I think just referring to `Serializer` here is probably better.